pkger/clone_resource.go

```diff
@@ -502,18 +502,14 @@ func ruleToResource(iRule influxdb.NotificationRule, endpointName, name string)
 }
 
 // regex used to rip out the hard coded task option stuffs
-var taskFluxRegex = regexp.MustCompile(`option task = {(.|\n)*}`)
+var taskFluxRegex = regexp.MustCompile(`option task = {(.|\n)*?}`)
```

Comment:
needed to make this not greedy

regex plagyround: https://regex101.com/r/xV9wL4/30